### DISCUSSION TITLE
Where is the code for the `${API_URL}/projects/${projectRef}/analytics/endpoints/logs.all?${queryParams}` endpoint?

### DISCUSSION THREAD

**Discussion by FelixZY** [supabase/supabase#11718] [Questions] [answered]
I'm looking to improve my API token system, perhaps even to create a PR if the implementation can be made good enough.

My research so far has led me to the endpoint `${API_URL}/projects/${projectRef}/analytics/endpoints/logs.all?${queryParams}`. However, I have been unsuccessful in finding the code responsible for it.

Anyone with more experience in the project who can point me in the right direction? 🙂

**Comment by GaryAustin1** [comment]
In general anything above the single project level is not part of the opensource repositories.  The logs are also not part of the public repository yet.

https://github.com/search?q=org%3Asupabase+analytics%2Fendpoints&type=code
Shows the only hits on that endpoint in all of Supabase.  I realize, not absolute confirmation.

From:  https://supabase.com/docs/guides/platform/logs

> note
> 
> These features are not currently available for self-hosting and local development.
> This is on the roadmap and you can follow the progress in the [Logflare repository](https://github.com/Logflare/logflare).

**Comment by Ziinc** [comment, marked as answer]
This API endpoint is part of the Supabase logging internal infrastructure and is hence the code is not publicly available. You can however use the endpoint as part of the management api, though there is no guarantees that this api endpoint will not change in the future.
